c7n/policy.py

```diff
@@ -465,9 +454,10 @@ def provision(self):
                 "Provisioning policy lambda %s", self.policy.name)
             variables = {
                 'account_id': self.policy.options.account_id,
-                'policy': self.policy.data
+                'policy': self.policy.data,
+                'region': self.policy.options.region or 'default'
```

Comment:
not sure what the or 'default' bit is about, there isn't really a default value here. i removed it and tests pass fine.

Reply:
I was following the lead of this bit of code [here](https://github.com/capitalone/cloud-custodian/blob/db7c4718f2fdde54e0559869e4149b2725061747/c7n/policy.py#L225). 

So if `self.policy.options.region` always has a value then I can remove it.

Reply:
yeah.. it should always have a value, i see what you mean though, i think that default there was for testing, in practice when executing we're always targeting a known region which is set before we get to this code.